packages/adminrouter/extra/src/README.md

```diff
@@ -312,17 +312,18 @@ order to support the `/service` URL path prefix clients are using.
 
 ### Intended use
 It is important to remember that the `/service` endpoint was designed to work
-with root Marathon only. Tasks launched by Marathon-on-Marathon instances will
+with Root Marathon only. Tasks launched by Marathon-on-Marathon instances will
 not be reachable via it.
 
 In the case of Mesos frameworks, this becomes a bit more complex - all the
 frameworks are visible in Mesos' `/state-summary` endpoint output so in theory
 it is possible to access a framework launched using Marathon-on-Marathon or by
-hand.  Unfortunately, Mesos as of now does not enforce framework name
-uniqueness and thus it is possible to launch multiple frameworks with the same
-name. AR in such case will route in a non-deterministic manner. Root Marathon
-by default enforces unique names for all the tasks and frameworks it has under
-control.
+hand (except in DC/OS Enterprise where only frameworks that are started by Root 
+Marathon can be routed to).  Unfortunately, Mesos as of now does not enforce 
```

Comment:
> except in DC/OS Enterprise where only frameworks that are started by Root Marathon can be routed to).
Doesn't it apply to OSS as well? Also, I hope that you will not mind that I throw my 2 cents here and ask if we can clarify where this limitation stems from (i.e. mesos roles).

Reply:
Pawel in DC/OS Enterprise Admin Router sees the filtered state-summary, whereas in in DC/OS it sees the complete state-summary.